tests/test.local.sh

```
@@ -1 +1 @@
-./node_modules/.bin/cypress run --headed --spec cypress/integration/admin/08-add-api-categories.spec.js
\ No newline at end of file
+./node_modules/.bin/cypress run --headed --spec cypress/e2e/admin/08-add-api-categories.cy.js
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)